docs/admin/multi_tenancy.md

```diff
@@ -70,227 +70,6 @@ The credentials used to log in to the {{site.data.keyword.ieam}} console are the
-   hzn exchange agbot addpattern IBM/agbot IBM '*' $NEW_ORG_ID
-   hzn exchange org list $NEW_ORG_ID
-   ```
-   {: codeblock}
-
-5. Assign the organization admin user role to the IAM user you previously chose and verify the user was created in the {{site.data.keyword.ieam}} exchange with the organization admin role:
-
-   ```bash
-   hzn exchange user create -o $NEW_ORG_ID -A $ORG_ADMIN_USER "" "<email-address>"
-   hzn exchange user list -o $NEW_ORG_ID $ORG_ADMIN_USER
-   ```
-   {: codeblock}
-
-   In the listing of the user, you should see: `"admin": true`
-
-**Note:** If you create multiple organizations and want the same organization admin to manage all of the organizations, use the same value for `ORG_ADMIN_USER` each time through this section.
-
-The organization admin can now use the [{{site.data.keyword.ieam}} management console](../console/accessing_ui.md) to manage {{site.data.keyword.ieam}} resources within this organization.
-
-### Enabling the organization admin to use the CLI
-
-In order for an organization admin to use the `hzn exchange` command to manage {{site.data.keyword.ieam}} resources using the CLI, they must:
-
-1. Use `cloudctl` to log in to the {{site.data.keyword.ieam}} management hub and create an API key:
-
-   ```bash
-   cloudctl login -a <cluster-url> -u $ORG_ADMIN_USER -p <hub-admin-password> -c $IAM_ACCOUNT_ID --skip-ssl-validation
-   cloudctl iam api-key-create "${ORG_ADMIN_USER}-api-key" -d "API key for $ORG_ADMIN_USER"
-   ```
-   {: codeblock}
-
-   Find the API key value in the command output line that starts with **API Key**. Save the key value in a safe location for future use because you cannot query it from the system later. Also, set it in this variable for the subsequent commands:
-
-   ```bash
-   ORG_ADMIN_API_KEY=<IAM API key you just created>
-   ```
-   {: codeblock}
-
-   **Tip:** If you add this user to additional IAM accounts in the future, you do not need to create an API key for each account. The same API key will work in all IAM accounts this user is a member of, and therefore in all {{site.data.keyword.ieam}} organizations this user is a member of.
-
-2. Verify the API key works with the `hzn exchange` command:
-
-   ```bash
-   export HZN_ORG_ID=<organization id>
-   export HZN_EXCHANGE_USER_AUTH=root/iamapikey:$ORG_ADMIN_API_KEY
-   hzn exchange user list
-   ```
-   {: codeblock}
-
-
-The new organization is ready to use. To set a maximum number of edge nodes in this organization, or customize the default edge node heartbeat settings, see [Organization Configuration](#org-config).
-
-## Non-admin users within an organization
-{: #org-users}
-
-A new user can be added to an organization by importing and onboarding the IAM user (as a `MEMBER`) to the corresponding IAM account. You do not need to explicitly add the user to the {{site.data.keyword.ieam}} exchange, because that occurs automatically when necessary.
```

Comment:
You can add a new user an organization by importing and onboarding the IAM user (as a `MEMBER`) to the corresponding IAM account. You do not need to explicitly add the user to the {{site.data.keyword.ieam}} exchange, because that occurs automatically when necessary.

Question: in the second sentence...would it be useful to give an example of "when necessary?"